__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => {
+  return db.end();
+});
+beforeEach(() => seed(testData));
+
+describe("GET /api/healthcheck", () => {
+  test("return 200 status code", () => {
+    return request(app).get("/api/healthcheck").expect(200);
+  });
+});
+
+describe("GET /api", () => {
+  test("return 200 status code", () => {
+    return request(app).get("/api").expect(200);
+  });
+  test("return an array of endpoints", () => {
+    return request(app).get("/api").expect(200).then(({body}) => {
+      console.log(body.endpoints);
+      expect("GET /api" in body.endpoints).toBeTruthy();
+      expect("GET /api/topics" in body.endpoints).toBeTruthy();
+      for (const endpointKey in body.endpoints) {
```

Comment:
we will add endpoints as we go along, can you think of a way to make this more dynamic without having to change the test every time?